src/registry.ts

```diff
@@ -1,6 +1,6 @@
 import {SignifyClient} from "signify-ts";
 
 export const QVI_CREDENTIAL_REGISTRY = "qvi_credential_registry";
-export const registry = async (client: SignifyClient, name: string) => {
+export const getQviRegistry = async (client: SignifyClient, name: string) => {
```

Comment:
`get` seems redundant to me and this is the `qvi` library so that seems redundant too.

Reply:
i think if you just name it registry, might be a little to vague? are you creating a registry? updating an entry in it? retrieveing it? I don't think it hurts to be a little verbose

Reply:
`getRegistry` then